math/big/summer_external_test.go

```diff
@@ -39,7 +39,7 @@ func TestSummer(t *testing.T) {
 	for i := 0; i < 10; i++ {
 		a, b := randomSummer(i, rng), randomSummer(i+1, rng)
 		equal, err := polybig.EqualSum(a, b)
-		require.EqualError(t, err, "dimension mismatch")
+		require.True(t, polybig.IsErrDimensionMismatch(err))
```

Comment:
There's [`require.ErrorIs`](https://pkg.go.dev/github.com/stretchr/testify@v1.7.0/require#ErrorIs) for that.

Reply:
Yes but i also want to test the implementation of `IsErrDimensionMismatch`, since it does not have a dedicated test curreently.